SignalMessaging/profiles/OWSProfileManager.m

```diff
@@ -1146,33 +1146,27 @@ - (void)updateProfileAvatarCache:(nullable UIImage *)image filename:(NSString *)
     }
 }
 
-+ (NSString *)legacyProfileAvatarsDirPath
+- (NSString *)mainAppProfileAvatarsDirPath
 {
     return [[OWSFileSystem appDocumentDirectoryPath] stringByAppendingPathComponent:@"ProfileAvatars"];
 }
 
-+ (NSString *)sharedDataProfileAvatarsDirPath
+- (NSString *)sharedDataProfileAvatarsDirPath
 {
     return [[OWSFileSystem appSharedDataDirectoryPath] stringByAppendingPathComponent:@"ProfileAvatars"];
```

Comment:
So profiles are copied into the group container for the share extension?

Reply:
Not yet - but they will be. I've made a card.  I've also made cards around copying over the `SignalAccount` and `TSThread` to a database in the shared data container, and possibly some other state too.